bin/backup.sh

```diff
@@ -0,0 +1,132 @@
+
+  # Get the values
+  mongoexport -o classifications_cset_values.csv --uri=${MONGODB_URI} -v --query='{ "namespace": "CSET", "publish": true }' --type=csv --collection=classifications --noHeaderLine --fields='_id,incident_id,namespace,publish,attributes.0.value_json,attributes.1.value_json,attributes.2.value_json,attributes.3.value_json,attributes.4.value_json,attributes.5.value_json,attributes.6.value_json,attributes.7.value_json,attributes.8.value_json,attributes.9.value_json,attributes.10.value_json,attributes.11.value_json,attributes.12.value_json,attributes.13.value_json,attributes.14.value_json,attributes.15.value_json,attributes.16.value_json,attributes.17.value_json,attributes.18.value_json,attributes.19.value_json,attributes.20.value_json,attributes.21.value_json,attributes.22.value_json,attributes.23.value_json,attributes.24.value_json,attributes.25.value_json,attributes.26.value_json,attributes.27.value_json,attributes.28.value_json,attributes.29.value_json,attributes.30.value_json,attributes.31.value_json'
+
+  # Construct the header
+  echo -n "_id,incident_id,namespace,publish," > tmp.csv
+  head -n 1 classifications_cset_headers.csv > tmp_header.csv
+  cat tmp.csv tmp_header.csv > header.csv
+
+  # Concat the header and the values to the output
+  cat header.csv classifications_cset_values.csv > ${TARGET}/classifications_cset.csv
+
+  # Cleanup
+  rm tmp.csv
+  rm tmp_header.csv
+  rm header.csv
+  rm classifications_cset_headers.csv
+  rm classifications_cset_values.csv
+
+else # Private backup
+  echo "Dump MongoDB 'aiidprod' database..."
+  mongodump -o ${TARGET} --uri=${MONGODB_URI} 
+
+  echo "Dump MongoDB 'translations' database..."
+  mongodump -o ${TARGET} --uri=${MONGODB_URI_TRANSLATIONS}
+fi
+
+echo "Report contents are subject to their own intellectual property rights. Unless otherwise noted, the database is shared under (CC BY-SA 4.0). See: https://creativecommons.org/licenses/by-sa/4.0/" > ${TARGET}/license.txt
+
+# run tar command
+echo "Start backup ${TARGET} into ${TARGET_BUCKET_URL} ..."
```

Comment:
can you review how `TARGET_BUCKET_URL` is configured in our workflow? It seems that this is set to `vars.AWS_S3_{PRIVATE|PUBLIC}_BUCKET`, so its usage here is misleading if we are copying to R2.

Reply:
I kept the `TARGET_BUCKET-URL` name for the AWS S3 backups. Maybe is a bit confusing.
For R2 we use different variables, all of them with prefix `CLOUDFLARE_`. Cloudflare doesn't use a URL for backups.
The variables that we should set are:
- `CLOUDFLARE_ACCOUNT_ID`
- `CLOUDFLARE_R2_ACCESS_KEY`
- `CLOUDFLARE_R2_SECRET_KEY`
- `CLOUDFLARE_R2_PUBLIC_BUCKET`
- `CLOUDFLARE_R2_PRIVATE_BUCKET`